startup/BMM/workspace.py

```diff
@@ -60,8 +60,11 @@ def initialize_workspace():
 def check_profile_branch():
     here = os.getcwd()
     STARTUP = os.path.dirname(BMM.functions.__file__)
-    os.chdir(os.path.join(os.getenv('HOME'), '.ipython', 'profile_collection'))
-    branch = subprocess.check_output(['git', 'branch', '--show-current']).decode("utf-8")[:-1]
+    os.chdir(get_ipython().profile_dir.location)
+    try:
+        branch = subprocess.check_output(['git', 'branch', '--show-current']).decode("utf-8")[:-1]
+    except subprocess.CalledProcessError:
+        branch = "not a git repository"
```

Comment:
As we all have shared home directories, I clone `profile_collection` repos of the beamlines as `profile_collection_bmm`, etc. This addition would handle the following error happening on such cloned repos (on today's version of the `master` branch):
```py
20:03 $ BS_ENV=collection-2021-1.2 BS_PROFILE=collection_bmm bsui

Versions of DSSI software:

    - Bluesky    : v1.6.7
    - Ophyd      : v1.6.1
    - Databroker : v1.2.2

Links to Bluesky and Databroker tutorials:

    - http://nsls-ii.github.io/bluesky/tutorial.html
    - http://nsls-ii.github.io/databroker/tutorial.html

If you get lost or confused, then we want to know! We have a friendly chat channel:

    https://gitter.im/NSLS-II/DAMA

or you can file a bug to let us know where our documentation could be made more clear:

    https://github.com/NSLS-II/Bug-Reports/issues

bsui is running these commands now to start an interactive computing environment for data acquisition:
    $ conda activate collection-2021-1.2
    $ PYTHONPATH=/nsls2/users/mrakitin/src/mrakitin/SRW/env/work/srw_python:
    $ ipython --profile=collection_bmm --IPCompleter.use_jedi=False

Python 3.7.7 (default, Mar 26 2020, 15:48:22)
Type 'copyright', 'credits' or 'license' for more information
IPython 7.21.0 -- An enhanced Interactive Python. Type '?' for help.

IPython profile: collection_bmm
environment variable BLUESKY_LOG_FILE is not set, using default log file path '/nsls2/users/mrakitin/.cache/bluesky/log/bluesky.log'
environment variable BLUESKY_IPYTHON_LOG_FILE is not set, using default file path '/nsls2/users/mrakitin/.cache/bluesky/log/bluesky_ipython.log'
Activating auto-logging. Current session state plus future input saved.
Filename       : /nsls2/users/mrakitin/.cache/bluesky/log/bluesky_ipython.log
Mode           : append
Output logging : True
Raw input log  : False
Timestamping   : True
State          : active
Exception reporting mode: Minimal
Checking workspace on this computer ...
fatal: not a git repository (or any parent up to mount point /nsls2/users)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
CalledProcessError: Command '['git', 'branch', '--show-current']' returned non-zero exit status 128.


In [1]:
```

So, it's useful not only for CI, but also for regular (not xf*) users.